Sources/RichTextKit/Attributes/RichTextAttributeWriter+Font.swift

```diff
@@ -93,7 +93,7 @@ private extension RichTextAttributeWriter {
     /// We must adjust empty font names on some platforms.
     func settableFontName(for fontName: String) -> String {
         #if macOS
-        fontName.isEmpty ? "Helvetica" : fontName
+        fontName
```

Comment:
We need to make sure the fontName is never empty. Although I didn't run into any crashes with this.

Reply:
If this works, the entire settable font name should be removed. I had some problems with the standard fonts on macOS and iOS. But perhaps the other fixes below fixes that.